### ISSUE TITLE
scam.

### ISSUE THREAD

**Issue by htkcodes** [hukhho/PancakeSwap-Front-Run-Contract-Bot#1]
**Nothing is free. if it was free everyone would be doing this.. this guy is just trying to scam you.**

**Comment by Decemberfest** [comment]
+

**Comment by hukhho** [comment]
yes, this code and all others code in youtube and github are scam. me too

**Comment by hukhho** [comment]
> **Nothing is free. if it was free everyone would be doing this.. this guy is just trying to scam you.**

yes. sorry for that. all code in youtube scam too

**Comment by UncoverScams** [comment]
@hukhho Documented here: https://twitter.com/UncoverScams/status/1468665394357542912 and you are welcome.
